app/javascript/__tests__/pages/form/components/Select.test.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react"
+import { useForm } from "react-hook-form"
+import { render, screen } from "@testing-library/react"
+import { userEvent } from "@testing-library/user-event"
+import { t } from "@bloom-housing/ui-components"
+import { FormStepProvider } from "../../../../formEngine/formStepContext"
+import { FormEngineProvider } from "../../../../formEngine/formEngineContext"
+import { openRentalListing } from "../../../data/RailsRentalListing/listing-rental-open"
+import FormSelect from "../../../../pages/form/components/Select"
+
+const FormSelectWrapper = () => {
```

Comment:
@jimlin-sfgov added unit test - will replace this wrapper with T's [util function](https://github.com/SFDigitalServices/sf-dahlia-web/pull/2723/files#diff-6453e399a136b04c7e57eed9087e0d21965e57530f7918be8bf937e72a758a08) once that is merged